UnitsNet.Serialization.JsonNet.Tests/UnitsNetJsonConverterBaseTests.cs

```diff
@@ -0,0 +1,113 @@
+﻿// Copyright Initial Force AS.  All rights reserved.
+// https://github.com/anjdreas/UnitsNet
+// 
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+// 
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+// 
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+
+using System;
+using Newtonsoft.Json;
+using NUnit.Framework;
+
+namespace UnitsNet.Serialization.JsonNet.Tests
+{
+    [TestFixture]
+    public class UnitsNetJsonConverterBaseTests
+    {
+        private JsonSerializerSettings _jsonSerializerSettings;
+        private UnitsNetJsonConverterBase<Mass> _massJsonConverter;
+        private UnitsNetJsonConverterBase<Ratio> _ratioJsonConverter;
+
+        [SetUp]
+        public void Setup()
+        {
+            _massJsonConverter = new MassJsonConverter();
+            _ratioJsonConverter = new RatioJsonConverter();
+
+            _jsonSerializerSettings = new JsonSerializerSettings() { Formatting = Formatting.Indented };
+            _jsonSerializerSettings.Converters.Add(_massJsonConverter);
```

Comment:
It would be ideal if we could do with just a single unitsnet converter here, but I'm fine with this for now. We can always take a stab at that in a future PR.